kernel/kata_config_version

```diff
@@ -1 +1 @@
-72
+73
```

Comment:
Nit: I don't think this is strictly necessary since you are not changing any (other) files below `kernel/*` here.

Reply:
ok, I will change it back.